trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestMissingStationArrivalDistanceService.java

```diff
@@ -0,0 +1,61 @@
+package io.edpn.backend.trade.application.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.edpn.backend.messageprocessorlib.application.dto.eddn.data.StationDataRequest;
+import io.edpn.backend.trade.application.domain.Message;
+import io.edpn.backend.trade.application.domain.filter.FindStationFilter;
+import io.edpn.backend.trade.application.dto.web.object.mapper.MessageMapper;
+import io.edpn.backend.trade.application.port.outgoing.kafka.SendKafkaMessagePort;
+import io.edpn.backend.trade.application.port.outgoing.station.LoadStationsByFilterPort;
+import io.edpn.backend.trade.application.port.outgoing.stationarrivaldistancerequest.CreateStationArrivalDistanceRequestPort;
+import io.edpn.backend.trade.application.port.outgoing.stationarrivaldistancerequest.RequestMissingStationArrivalDistanceUseCase;
+import io.edpn.backend.util.Module;
+import io.edpn.backend.util.Topic;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.retry.support.RetryTemplate;
+import org.springframework.scheduling.annotation.Scheduled;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+
+@AllArgsConstructor
+@Slf4j
+public class RequestMissingStationArrivalDistanceService implements RequestMissingStationArrivalDistanceUseCase {
+
+    public static final FindStationFilter FIND_STATION_FILTER = FindStationFilter.builder()
+            .hasArrivalDistance(false)
+            .build();
+
+    private final LoadStationsByFilterPort loadStationsByFilterPort;
+    private final CreateStationArrivalDistanceRequestPort createStationArrivalDistanceRequestPort;
+    private final SendKafkaMessagePort sendKafkaMessagePort;
+    private final RetryTemplate retryTemplate;
+    private final Executor executor;
+    private final ObjectMapper objectMapper;
+    private final MessageMapper messageMapper;
+
+    @Override
+    @Scheduled(cron = "0 0 */12 * * *")
```

Comment:
Not a valid cron job?
Unless theres some funky java syntax every 12 hours is: 0 */12 * * *
drop the leading 0